testsuite/gnat2goto/tests/declaration_rename_N_Renaming_Declaration/declaration_rename.adb

```diff
@@ -18,7 +18,7 @@ procedure Declaration_Rename is
    function "=" (Left, Right: ET.New_Integer) return Boolean renames ET."=";
 
    --  currently unsupported --
```

Comment:
Can you remove this comment please? It’s apparently incorrect now.

Reply:
Yes!  Please do!